src/test/kotlin/uk/gov/justice/digital/hmpps/externalusersapi/resource/UserControllerIntTest.kt

```diff
@@ -520,4 +520,69 @@ class UserControllerIntTest : IntegrationTestBase() {
         .jsonPath("$.[*].groupCode").value<List<String>> { assertThat(it.size > 2) }
     }
   }
+
+  @Nested
+  inner class FindUserByUserId {
```

Comment:
rename if you've renamed the controller method

Reply:
Changed, though feel free to comment on the name again as not sure about it.